File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/member/SecurityLevel.java
package com.example.cf_sdk.logic.changebanksdk.model.member;

import com.google.gson.annotations.SerializedName;

/**
 *
 * Enum for the Security Level.
 */

public enum SecurityLevel {

    @SerializedName("HIGH")
    HIGH("High"),

    @SerializedName("MEDIUM")
    MEDIUM("Medium");

    String mLevelName;

    SecurityLevel(String levelName) {
        mLevelName = levelName;
    }

    public String getLevelName() {
        return mLevelName;
    }

}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/member/SecurityLevel.kt
package com.example.sdk_no_dagger.changebankapi.model.member

import com.google.gson.annotations.SerializedName

/**
 *
 * Enum for the Security Level.
 */
enum class SecurityLevel(levelName: String) {
    @SerializedName("HIGH")
    HIGH("High"), @SerializedName("MEDIUM")
    MEDIUM("Medium");

    var levelName = ""

    init {
        this.levelName = levelName
    }
}